customizations/generator/openapi/src/main/resources/templates/expediagroup-sdk/client.mustache

```diff
@@ -26,12 +26,27 @@ class {{clientClassname}}Client private constructor(clientConfiguration: RapidCl
 
     class Builder : BaseRapidClient.Builder<Builder>() {
         override fun build() = {{clientClassname}}Client(
-            RapidClientConfiguration(key, secret, endpoint, requestTimeout, connectionTimeout, socketTimeout, maskedLoggingHeaders, maskedLoggingBodyFields)
+            RapidClientConfiguration(key, secret, endpoint, requestTimeout, connectionTimeout, socketTimeout, maskedLoggingHeaders, maskedLoggingBodyFields, null)
         )
     }
 
+    class BuilderWithHttpClient() : BaseRapidClient.BuilderWithHttpClient<BuilderWithHttpClient>() {
+        override fun build() : {{clientClassname}}Client {
+
+            if (okHttpClient == null) {
```

Comment:
I don't think this is needed...

Reply:
We agreed to raise an exception if the okhttpclient is not passed when using this builder